### ISSUE TITLE
[Enhancement request] Avoid "That username is already taken"

### ISSUE THREAD

**Issue by kuraga** [doccano/doccano#738]
Enhancement description
---------

I have these errors in log on each start:

```
backend_1   | Initializing database
backend_1   | Operations to perform:
backend_1   |   Apply all migrations: admin, api, auth, authtoken, contenttypes, sessions, social_django
backend_1   | Running migrations:
backend_1   |   No migrations to apply.
backend_1   |   Your models have changes that are not yet reflected in a migration, and so won't be applied.
backend_1   |   Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
backend_1   | CommandError: Error: That username is already taken.
```

Analysis
-----------

The error
```
backend_1   | CommandError: Error: That username is already taken.
```
comes from here: 
https://github.com/doccano/doccano/blob/bdba2961bf7942eb529a1591a7499b5d15af73bf/app/server/management/commands/create_admin.py#L22
i.e. here: 
https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L144-L146

Note 
https://github.com/doccano/doccano/blob/bdba2961bf7942eb529a1591a7499b5d15af73bf/app/server/management/commands/create_admin.py#L23
*doesn't* catch this `CommandError` and `handle`'s execution *is interrupted*.

**Comment by icoxfog417** [comment]
solved #787

**Comment by adhamhassan99** [comment]
How did you solve the issue i have the same error please help.
